packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss

```
@@ -162,8 +184,11 @@
       position: absolute;
       top: 0;
       right: 0;
-      width: calc(2 * var(--#{$block-class}--size));
+      display: flex;
+      width: calc(3 * var(--#{$block-class}--size));
```

This comment seems to have become detached from the line it refers to...

```suggestion
      // width: room for validation, and two buttons
      width: calc(3 * var(--#{$block-class}--size));
```